packages/mdc-menu/README.md

```diff
@@ -147,198 +122,101 @@ To disable a list item, set `aria-disabled` to `"true"`, and set `tabindex` to `
- - y1: The y value of control point P1.
- - x2: The x value of control point P2.
- - y2: The y value of control point P2.
-```
+  // Instantiation
+  var menuEl = document.querySelector('#toolbar');
+  var menu = new mdc.menu.MDCSimpleMenu(menuEl);
+  var menuButtonEl = document.querySelector('#menu-button');
+  
+  // Toggle menu open
+  menuButtonEl.addEventListener('click', function() {
+    menu.open = !menu.open;
+  });
+  
+  // Listen for selected item
+  menuEl.addEventListener('MDCSimpleMenu:selected', function(evt) {
+     var detail = evt.detail;
+  });
+  
+  // Set Anchor Corner to Bottom End
+  menu.setAnchorCorner(Corner.BOTTOM_END); 
+```
+
+### `MDCSimpleMenu`
+
+See [Importing the JS component](../../docs/importing-js.md) for more information on how to import JavaScript.
+
+Property | Value Type | Description
+--- | --- | ---
+`open` | Boolean | Proxies to the foundation's `isOpen`/(`open`, `close`) methods. 
+`items` | Array<Element> | Proxies to the foundation's container to query for all `.mdc-list-item[role]` elements.
+`itemsContainer` | Element | Queries the foundation's root element for the `mdc-simple-menu__items` container element.
+
+Method Signature | Description
+--- | ---
+`show({focusIndex: ?number}) => void` | Proxies to the foundations `open()` method. Optional options object containing a  `focusIndex` parameter to specify the item to receive focus after the menu opens.
+`hide() => void` | Proxies to the foundation's `close()` method.
+`setAnchorCorner(Corner) => void` | Proxies to the foundation's `setAnchorCorner(Corner)` method.
+`setAnchorMargin(AnchorMargin) => void` | Proxies to the foundation's `setAnchorMargin(AnchorMargin)` method.
+`getDefaultFoundation() => MDCSimpleMenuFoundation` | Returns the foundation. 
+
+### `MDCSimpleMenuAdapter`
+
+Method Signature | Description
+--- | ---
+`addClass(className: string) => void` | Adds a class to the root element.
+`removeClass(className: string) => void` | Removes a class from the root element.
+`hasClass(className: string) => boolean` | Returns boolean indicating whether element has a given class.
+`hasNecessaryDom() => boolean` | Returns boolean indicating whether the necessary DOM is present (namely, the `mdc-simple-menu__items` container).
+`getAttributeForEventTarget(target: EventTarget, attributeName: string) => string` | Returns the value of a given attribute on an event target.
+`eventTargetHasClass: (target: EventTarget, className: string) => boolean` | Returns true if the event target has a given class.
+`getInnerDimensions() => {width: number, height: number}` | Returns an object with the items container width and height.
+`hasAnchor: () => boolean` | Returns whether the menu has an anchor for positioning.
+`getAnchorDimensions() => { width: number, height: number, top: number, right: number, bottom: number, left: number }` | Returns an object with the dimensions and position of the anchor (same semantics as `DOMRect`).
```

Comment:
Nit: Remove spaces after `{` and before `}` (e.g., `=> {width: ..., left: number}`)